### PR TITLE
https://jira.fingo.info/browse/IJ-222 Fix removal of holidays from the previous year

### PR DESCRIPTION
**Pull request by boavenn**

### CODE REVIEW

File: view.react/src/components/holidays-config/HolidaysConfig.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const HolidaysConfig = () => {
     const {holidays} = state
 
     const currentYear = (new Date()).getFullYear();
-    const availableYears = [currentYear, currentYear + 1];
+    const availableYears = getAvailableYears(currentYear);
```
Comment (DaDudek) [inline view.react/src/components/holidays-config/HolidaysConfig.jsx:19]: i think its good idea to change now default year on date picker -> now its always current year, so maybe we change it to choosen year?

Comment (boavenn) [reply view.react/src/components/holidays-config/HolidaysConfig.jsx:19]: This function only generates possible years a user may choose from and `currentYear` is needed to generate correct values. The `selectedYear` field below dictates which one is chosen and displayed on the `YearPicker`

Comment (DaDudek) [reply view.react/src/components/holidays-config/HolidaysConfig.jsx:19]: i see say this in confusing way -> this comment is not about this line, these remarks do not apply to this line, only changes to this task. If you want add new holiday datepicker has default date ->first day of currentYear. Now when we have possibility to move more than one year back and add some holiday it is inconvenient to click through so many times changing the month (you can only move by one month, not by year). So my proposition is to change this mechanism. Now its work in that way:

1. if any date was selected, it was remembered until the card was refreshed 
2. if date wasn't selected first day of current year is choose

In my opinion we should change point 2. to something like this:
2. if date wasn't selected first day of selected year is choose

Comment (DaDudek) [reply view.react/src/components/holidays-config/HolidaysConfig.jsx:19]: from what i see we already pass the selected_year -> not current one. So its probably harder change than i thought it be. So we can add this as other task or avoid this. If you agree with one of this two option you can resolve this thread and consider this PR as accepted and tested

Comment (boavenn) [reply view.react/src/components/holidays-config/HolidaysConfig.jsx:19]: As concluded in the talk on slack - it's not an urgent or even reasonable change - so I'm resolving this thread

**Pull request merged**